zds/tutorialv2/models/database.py

```diff
@@ -151,6 +151,7 @@ class Meta:
     js_support = models.BooleanField("Support du Javascript", default=False)
 
     is_obsolete = models.BooleanField("Est obsolète", default=False)
+    obsolete_justif = models.TextField("Justificatif d'obsolescence", null=True, blank=True)
```

Comment:
Sans doute qu'on peut maintenant se passer de `is_obsolete`, puisque pour savoir si le contenu est obsolète, il suffit de tester si le message d'obsolescence n'est pas nul :
```suggestion
    obsolete_message = models.TextField("Justificatif d'obsolescence", null=True, blank=True)
```